token-locker/src/lib.rs

```diff
@@ -50,8 +50,8 @@ enum StorageKey {
 pub struct Contract {
     /// The account of the prover that we can use to prove.
     pub prover_account: AccountId,
-    /// Ethereum address of the token bridge contract, in hex.
-    bridge_address: EthAddress,
+    /// Ethereum address of the token factory contract, in hex.
+    factory_address: EthAddress,
```

Comment:
Let's call it `eth_factory_address` to make it more clear

Reply:
Fixed c112a8c38f43c876660c52e06a601a6ee01281f6